lib/data/wrapper.js

```diff
@@ -96,31 +96,36 @@ const data = {
         // If objectGetInfo.key exists the md-model-version is 2 or greater.
         // Otherwise, the objectGetInfo is just the key string.
         const key = objectGetInfo.key ? objectGetInfo.key : objectGetInfo;
-        log.debug('sending delete to datastore', { implName, key,
+        log.debug('sending delete to dataStore', { implName, key,
             method: 'delete' });
         client.delete(key, log.getSerializedUids(), err => {
             if (err) {
-                log.error('error from sproxyd', { error: err });
+                log.error('error deleting object from dataStore',
+                    { error: err }, { location: key });
                 return cb(errors.InternalError);
             }
+            log.trace('delete object successful');
             return cb();
         });
     },
 
     // It would be preferable to have an sproxyd batch delete route to
     // replace this
-    batchDelete: (locations, log) => {
+    batchDelete: (locations, log, cb) => {
```

Comment:
Not related to this PR but `batchDelete` should be `multiDelete`. The former is more about to provide the atomicity (all or nothing deleted) while the latter could be anything (0-to-all deleted). It'll be good to use the right term.